### DISCUSSION TITLE
blog/how-can-i

### DISCUSSION THREAD

**Discussion by giscus** [TkDodo/blog-comments#82] [Announcements]
# blog/how-can-i

Asking the right questions is a form of art that needs to be mastered.

https://tkdodo.eu/blog/how-can-i#inject-comments

**Comment by sigfriedCub1990** [comment]
That's exactly the point of [this essay](https://www.catb.org/esr/faqs/smart-questions.html) by Eric S. Raymond, I always encourage my co-workers to read it in order to get better at asking questions.
